source/max-staleness/tests/ReplicaSetNoPrimary/Unavailable.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Why is this test named `Unavailable.json`? That would seem to indicate that there are no available servers. But this test includes 2 servers that should be ignored and one that can service the request. So why `Unavailable.json`?